### ISSUE TITLE
Compiling Apollo nowaday using last ps3toolchain

### ISSUE THREAD

**Issue by crystalct** [bucanero/apollo-ps3#47]
Hi bucanero, i just compiled Apollo using a fresh psl1ght installation.
I found this issues:
```
fatal error: libxml/parser.h: No such file or directory
 #include <libxml/parser.h>
```
just add `-I$(PORTLIBS)/include/libxml2` to Makefile so all compiling platforms will be compatible

```
fatal error: polarssl/sha256.h: No such file or directory
 #include <polarssl/sha256.h>
```
You should inform that we need to install polar 1.3.9 from ps3libraries develop branch

```
fatal error: audioplayer.h: No such file or directory
 #include "audioplayer.h"
```
just add `-I$(PORTLIBS)/include/soundlib` to Makefile so all compiling platforms will be compatible

```
fatal error: freetype/config/ftheader.h: No such file or directory
 #include <freetype/config/ftheader.h>
```
just add `-I$(PORTLIBS)/include/freetype2` to Makefile so all compiling platforms will be compatible

^_^

**Comment by bucanero** [comment]
ciao @crystalct, thanks for the build comments/notes

indeed, I'll try to update the `Makefile` and add those include folders so it can be built with a stock PSL1GHT setup. I have a few symlinks in my system (like `freetype2` -> `freetype`) that are not there by default and can bring these compile errors

about polarSSL 1.3.9, I should at least update the README and link to the develop branch, or my own ps3libraries fork that has the updated polarssl 1.3.9

grazie! 😄

**Comment by bucanero** [comment]
I've updated the `Readme.md` with a link to the updated polarSSL from my fork: https://github.com/bucanero/ps3libraries/blob/master/scripts/015-polarssl-1.3.9.sh

**Comment by bucanero** [comment]
hello @crystalct! 👋 

sorry for the delay, it took me some time but I just had to build the PS3 toolchain from scratch (new computer) and tested to build Apollo with "default" settings.

I have added to the makefile:
```
-I$(PORTLIBS)/include/freetype2
-I$(PORTLIBS)/include/libxm2
```

for `audioplayer.h`, I modified the .c source to `#include <soundlib/audioplayer.h>`

Everything should be fixed and working now, in 9d1348486351ed37aec9dbb5711378b91acfc6f2

**Comment by crystalct** [comment]
Cool!
